src/Pos/Constants.hs

```diff
@@ -26,7 +26,11 @@ module Pos.Constants
        , genesisN
        , genesisSlotDuration
        , genesisMaxBlockSize
+       , genesisMaxHeaderSize
+       , genesisMaxGetHeadersNum
        , genesisMaxTxSize
+       , genesisMaxReqSize
```

Comment:
Well, `genesisMaxGetHeadersNum, genesisMaxInvSize, genesisMaxReqSize` should be stripped of the prefix `genesis`. We use this prefix only for constants, that are periodically updated via US. But for these few this is not the case, AFAIU